internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -191,18 +182,13 @@ func (s *BackendService) Template() (string, error) {
 
 		// ALB configs.
 		ALBEnabled:          s.albEnabled,
-		Aliases:             aliases,
-		AllowedSourceIps:    allowedSourceIPs,
 		DeregistrationDelay: deregistrationDelay,
-		HostedZoneAliases:   hostedZoneAliases,
-		HTTPSListener:       s.httpsEnabled,
-		HTTPRedirect:        s.httpsEnabled,
 		HTTPTargetContainer: template.HTTPTargetContainer{
 			Port: targetContainerPort,
 			Name: targetContainer,
 		},
 		HTTPHealthCheck: convertHTTPHealthCheck(&s.manifest.RoutingRule.HealthCheck),
```

Comment:
Shall we move `HTTPTargetContainer` and `HTTPHealthCheck` to `ALB` as well? They seem to be unique per listener too?

Reply:
So the reason why we can not move this configuration inside ALB is: 
1. `HTTPTargetContainer` this is being used with service connect and if we put it inside ALB then it would be more complicated to figure out which configuration to be used for Service Connect from the list of listener rules having `HTTPSTargetContainer`. So I kept this config as is until we decide to support multiple ports through SC. 
2. For `HTTPHealthCheck` Wanxian also asked the same question [here](https://github.com/aws/copilot-cli/pull/4501#discussion_r1126997091). So we have a bug where we use `GracePeriod` param from HTTPSHealthCheck for ALB as well as NLB. So we will fix this bug separately.